src/index.ts

```diff
@@ -52,6 +53,17 @@ const testStore: GroupedTests = {
   },
 };
 
+const logTime = (startTime: number, endTime: number, options: Options) => {
+  log(
+    `Finished: After script in ${
```

Comment:
This log statement should be outside the function, only the math part should be kept here.

Otherwise after every test, after and before block - the same log statement will be printed, i.e. Finished: `After` script in

![Screenshot 2022-10-15 at 2 55 16 PM](https://user-images.githubusercontent.com/4240966/195979277-f88cb084-1411-4fb7-8027-552ab5c67fe7.png)